exec/plugin-commands-script-runners/src/regexp-command.ts

```diff
@@ -0,0 +1,15 @@
+export function buildRegExpFromCommand (
```

Comment:
we use camelCase to name files.

Reply:
Sorry, just mistake 🙇 Thanks ! [671a063](https://github.com/pnpm/pnpm/pull/5871/commits/671a063f899668b65a70650b82599619ecb08ddf)